src/Vizkit3dWorld.cpp

```diff
@@ -155,6 +164,14 @@ vizkit3d::RobotVisualization* Vizkit3dWorld::robotVizFromSdfModel(sdf::ElementPt
     //vizkit3d plugin with model defined in the sdf model
     vizkit3d::RobotVisualization* robotViz = new vizkit3d::RobotVisualization();
 
+    for (int i=0; i < excludeFromSdf.size(); ++i){
```

Comment:
@doudou , I've tried to ignore the entire model, but it didn't work. I've got the following message.

![error](https://cloud.githubusercontent.com/assets/8384920/14616602/76bfed22-0581-11e6-8586-2afc1bae19af.png)

Since the links corresponds to a frame, it seems that I have to load at least them. Because I think Vizkit3d fails when it tries to find the frame names.